frontend/src/components/commons/InboxButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+const styles = {
+  button: {
+    width: 180,
+    marginBottom: 10,
+    marginLeft: 20,
+    marginRight: 20,
+    textAlign: "center",
+    borderRadius: 4,
+    padding: 6,
+    border: "2px solid #009fae"
+  }
+};
+
+class InboxButton extends Component {
```

Comment:
I think this can go in /emib/ and be called `EmailPreview`. It's not really a common component. You could also make an /emib/inbox/ folder if you think it'd be helpful.

Reply:
Refactored in 8520bbc